app/_hub/kong-inc/openid-connect/_index.md

```diff
@@ -974,7 +1028,25 @@ params:
         - `Lax`: Cookies are not sent on normal cross-site subrequests (for example to load images or frames into a third party site), but are sent when a user is navigating to the origin site (for example, when following a link).
         - `None`: Cookies will be sent in all contexts, for example in responses to both first-party and cross-origin requests. If SameSite=None is set, the cookie Secure attribute must also be set (or the cookie will be blocked)
         - `off`: Do not set the Same-Site flag.
+    - name: session_cookie_same_site
+      minimum_version: "3.2.x"
+      required: false
+      default: '"Lax"'
+      datatype: string
+      description: |
+        Controls whether a cookie is sent with cross-origin requests, providing some protection against cross-site request forgery attacks:
+        - `Strict`: Cookies will only be sent in a first-party context and aren't sent along with requests initiated by third party websites.
+        - `Lax`: Cookies are not sent on normal cross-site subrequests (for example to load images or frames into a third party site), but are sent when a user is navigating to the origin site (for example, when following a link).
+        - `None`: Cookies will be sent in all contexts, for example in responses to both first party and cross-origin requests. If SameSite=None is set, the cookie Secure attribute must also be set (or the cookie will be blocked)
+        - `Default`: Do not explicitly specify a Same-Site attribute.
```

Comment:
```suggestion
        - `Default`: Do not explicitly specify a `Same-Site` attribute.
```
Is it Same-Site or SameSite? The previous bullet point has it as SameSite.

Reply:
fixed it, thanks